test/performancetest/performance_test.go

```diff
@@ -51,20 +51,13 @@ func TestPerformance(t *testing.T) {
 	instanceId := awsservice.GetInstanceId()
 	log.Printf("Instance ID used for performance metrics : %s\n", instanceId)
 
-	configFilePath, logStreams, err := GenerateConfig(logNum)
+	configFilePath, err := GenerateConfig(logNum)
 	if err != nil {
 		t.Fatalf("Error: %v", err)
 	}
 
-	//defer deleting log group and streams
-	//defer deleting log group first because golang handles defers in LIFO order
-	//and we want to delete the log group after deleting the log streams
 	defer awsservice.DeleteLogGroup(instanceId)
```

Comment:
```
Does deleting a log group by default also delete any log streams in it?
```
Yes. I'm even not sure why we `DeleteLogGroupAndLogStream` to begin with. However, in the performance test. the LogStream are being generated dynamically. Therefore, if using  `DeleteLogGroupAndLogStream` it will create throttle on backend side. To simplify things, only use `DeleteLogGroup`